client/src/components/Card.js

```diff
@@ -1,48 +1,207 @@
 import React, { Component } from 'react';
 import '../styles/Card.css';
+import {
```

Comment:
nit: I don't think all of these imports are used here, as far as I can tell (Jumbotron for example). The same might be true for other components